tests/fsharp/core/measures/test.fsx

```diff
@@ -509,7 +509,8 @@ module MembersTest =
     let s = 2.0f<kg>
     let d = 2.0M<kg>
 
-#if !NetCore
+#if NetCore
+#else
```

Comment:
i got this time ago during the nunit testsuite pr, running test with old version. 
Maybe now is ok, sry for late response